dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Register/PreferredNamePage.cshtml.cs

```diff
@@ -23,7 +23,7 @@ public PreferredNamePage(SignInJourney journey)
     public string ExistingName => _journey.AuthenticationState.GetName()!;
 
     [BindProperty]
-    [Display(Name = "Use preferred name?")]
+    [Display(Name = " ")]
```

Comment:
For radio fieldset legend always visible despite no legend tag in the corresponding cshtml page. This slight workaround makes sure the legend doesn't render

Reply:
I think we need to alter the markup here so that we do have a legend (and the `<h1>` and `<p>` tags go into that rather being outside of the component).

Reply:
Not entirely sure what you mean here. Do you have an example?

Reply:
Something like: https://github.com/DFE-Digital/get-an-identity/blob/e8a2ffea6a8de027a08944330c81b338610485f9/dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Trn/AwardedQtsPage.cshtml#L15